server/src/middlewares/HandleUsersRequest.ts

```diff
@@ -0,0 +1,70 @@
+import { NextFunction, Request, Response } from 'express';
+
+import UserType from '../Models/User/UserType';
+import { DISTRICT_COUNTY_BY_USER } from '../DBService/Users/Query';
+import logger, { invalidDBResponseLog, launchingDBRequestLog, validDBResponseLog } from '../Logger/Logger';
+import { graphqlRequest, unauthorizedStatusCode } from '../GraphqlHTTPRequest';
+import { Severity } from '../Models/Logger/types';
+
+const handleUsersRequest = async (request: Request, response: Response, next: NextFunction) => {
+    const currentUser = response.locals.user;
+    const { userId } = request.body;
+
+    const usersMiddlewareLogger = logger.setup({
+        workflow: 'InvestigationMiddleware',
+        investigation: currentUser.epidemiologynumber,
+    });
+
+    const questionedUser = await getUserDistrictCounty(userId, response.locals);
+    if (currentUser.userType === UserType.SUPER_ADMIN) {
```

Comment:
'currentUser.userType' maybe move to const?

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1249/commits/51a1dbb29c72a734e562a38a45e089f0bea2498b)